ansible/vars/topo_dualtor-mux.yml

```diff
@@ -0,0 +1,157 @@
+topology:
+  dut_num: 2
+  host_interfaces:
+    - 0.0@0,1.0@0
+    - 0.1@1,1.1@1
+    - 0.2@2,1.2@2
+    - 0.3@3,1.3@3
+    - 0.4@4,1.4@4
+    - 0.5@5,1.5@5
+    - 0.6@6,1.6@6
+    - 0.7@7,1.7@7
+    - 0.8@8,1.8@8
+    - 0.9@9,1.9@9
+    - 0.10@10,1.10@10
+    - 0.11@11,1.11@11
+    - 0.12@12,1.12@12
+    - 0.13@13,1.13@13
+    - 0.14@14,1.14@14
+    - 0.15@15,1.15@15
+    - 0.16@16,1.16@16
+    - 0.17@17,1.17@17
+    - 0.18@18,1.18@18
+    - 0.19@19,1.19@19
+    - 0.20@20,1.20@20
+    - 0.21@21,1.21@21
+    - 0.22@22,1.22@22
+    - 0.23@23,1.23@23
+    - 0.24@24,1.24@24
+    - 0.25@25,1.25@25
+    - 0.26@26,1.26@26
+    - 0.27@27,1.27@27
+    - 0.28@28,1.28@28
+    - 0.29@29,1.29@29
+    - 0.30@30,1.30@30
+    - 0.31@31,1.31@31
+    - 0.32@32,1.32@32
+    - 0.33@33,1.33@33
+    - 0.34@34,1.34@34
+    - 0.35@35,1.35@35
+    - 0.36@36,1.36@36
+    - 0.37@37,1.37@37
+    - 0.38@38,1.38@38
+    - 0.39@39,1.39@39
+    - 0.40@40,1.40@40
+    - 0.41@41,1.41@41
+    - 0.42@42,1.42@42
+    - 0.43@43,1.43@43
+    - 0.44@44,1.44@44
+    - 0.45@45,1.45@45
+    - 0.46@46,1.46@46
+    - 0.47@47,1.47@47
+    - 0.48@48,1.48@48
+    - 0.49@49,1.49@49
+    - 0.50@50,1.50@50
+    - 0.51@51,1.51@51
+    - 0.52@52,1.52@52
+    - 0.53@53,1.53@53
+    - 0.54@54,1.54@54
+    - 0.55@55,1.55@55
+  disabled_host_interfaces:
+    - 0.1@1,1.1@1
+    - 0.3@3,1.3@3
+    - 0.5@5,1.5@5
+    - 0.7@7,1.7@7
+    - 0.9@9,1.9@9
+    - 0.11@11,1.11@11
+    - 0.13@11,1.13@11
+    - 0.15@11,1.15@11
+    - 0.17@17,1.17@17
+    - 0.19@19,1.19@19
+    - 0.21@21,1.21@21
+    - 0.23@23,1.23@23
+    - 0.25@25,1.25@25
+    - 0.27@27,1.27@27
+    - 0.29@29,1.29@29
+    - 0.31@31,1.31@31
+    - 0.33@33,1.33@33
+    - 0.35@35,1.35@35
+    - 0.37@37,1.37@37
+    - 0.39@39,1.39@39
+    - 0.41@41,1.41@41
+    - 0.43@43,1.43@43
+    - 0.45@45,1.45@45
+    - 0.47@47,1.47@47
+    - 0.49@49,1.49@49
+    - 0.51@51,1.51@51
+    - 0.53@53,1.53@53
+    - 0.55@55,1.55@55
+  DUT:
+    loopback:
+      ipv4:
+        - 10.1.0.32/32
+        - 10.1.0.33/32
+      ipv6:
+        - FC00:1::32/128
+        - FC00:1::33/128
+    loopback1:
+      ipv4:
+        - 10.1.0.34/32
+        - 10.1.0.35/32
+      ipv6:
+        - FC00:1::34/128
+        - FC00:1::35/128
+    loopback2:
+      ipv4:
+        - 10.1.0.36/32
+        - 10.1.0.36/32
+      ipv6:
+        - FC00:1::36/128
+        - FC00:1::36/128
+    loopback3:
+      ipv4:
+        - 10.1.0.38/32
+        - 10.1.0.39/32
+      ipv6:
+        - FC00:1::38/128
+        - FC00:1::39/128
+    vlan_configs:
+      default_vlan_config: one_vlan_a
+      one_vlan_a:
+        Vlan1000:
+          id: 1000
+          intfs: [0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 34, 36, 38, 40, 42, 44, 46, 48, 50, 52, 54]
+          prefix: 192.168.0.1/21
+          prefix_v6: fc02:1000::1/64
+          tag: 1000
+          mac: 00:aa:bb:cc:dd:ee
+      two_vlan_a:
```

Comment:
Not Sure, if this is vlan config is neccessary.

Reply:
Agree here, since we are using two DUTs we should only need a single VLAN to validate Y cable behavior